rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.2-release-notes.adoc

```diff
@@ -0,0 +1,32 @@
+= RPA Recorder 6.5.2 Release Notes
+
+RPA Recorder enables you to record a business process for automation.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== December , 2023
+
+// == Compatibility
+include::partial$release-notes/rn-headings.adoc[tag=compatibility,leveloffset=+1]
+
+[%header%autowidth.spread]
+|===
+|Application |Version
+|RPA Builder | 6.4.0 and later
+|===
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="2a,1a"]
+|===
+| Issue Resolution | ID
+|When recording a screenshot using the *Full screen* option in the *Define screenshot* pop-up, RPA Recorder no longer crashes when you resume recording or add an activity. |W-14573347
```

Comment:
The root of the problem here was the the Recording Toolbar was displayed during recording screenshots **manually**. The possibility to resume recording or to add an activity would not be available if the toolbar would not be displayed.
Therefore I suggest to combine both issues as something like "The Recording Toolbar is not displayed during adding manual screenshots anymore, until adding screenshot is finished".